Core/include/Acts/Seeding/HoughVectors.hpp

```diff
@@ -0,0 +1,161 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/Logger.hpp"
+
+#include <sstream>
+#include <stdexcept>
+#include <vector>
+
+template <typename T>
+class vector2D {
```

Comment:
This is a very generic name, and we have a `Vector2` already. Can we name this something that is more specific to what it's accomplishing?

Reply:
tried to find a nicer one!